02Lesson/02Project/index.html

```diff
@@ -1,10 +1,39 @@
 <!DOCTYPE html>
 <html>
-<head>
-  <title></title>
-  <link rel="stylesheet" href="css/style.css"/>
-</head>
-<body>
+  <head>
+    <title>DAILY PLANET</title>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="css/style.css" type="text/css"/>
+  </head>
+  <body>
+    <header>
+      <h1>
+        THE DAILY PLANET
+      </h1>
+      <nav>
+        <a href="http://www.statesman.com/sports/">SPORTS</a>
+        <a href="http://www.statesman.com/news/local/">LOCAL</a>
+        <a href="http://www.statesman.com/business/">ECONOMY</a>
+      </nav>
+    </header>
+    <main>
+      <section>
+        <article>
+          <h1>
+            Hillary Clinton’s campaign to run TV ads in Texas
+          </h1>
+          <p>
+            Hillary Clinton’s bla bla bla...

```

Comment:
Missing closing `p` tag. Make sure you don't have any errors in W3C validation...